node/container.go

```diff
@@ -502,6 +502,21 @@ func (a *HostAgent) getService(serviceID string) (*service.Service, error) {
 	return svc, nil
 }
 
+func(a *HostAgent) getServicePath(svc *service.Service) (string, error) {
+	var err error
+	s := svc
+	svc_path := fmt.Sprintf("/%s", svc.Name)
+	for s.ParentServiceID != "" {
+		s, err = a.getService(s.ParentServiceID)
```

Comment:
This will have negative impacts on performance for customers with 1000s of services.  I have a suggestion on how to fix, but will send it via direct email because it's more involved than I can explain in a code-review comment.   However, the short version is that we need to refactor the implementation of GetEvaluatedService() as well the associated caching, so that it also returns information on the service path.